### PR TITLE
finishes exercises

### PR DESCRIPTION
**Pull request by aylorbrown**
please review my code! my brain has is fully exercised!

### CODE REVIEW

**Comment by aylorbrown** [comment]
i thought i couldn't go any further but i finished the bonus exercises!! woo hoo!

File: scripts/index.js [hunk 1]
```diff
@@ -1 +1,179 @@
...
+console.log(evenNumbers([1, 2, 3, 4, 99, -90, 100, 25468]));
+
+// write function which takes an array of numbers as its argument and returns a new 
+// array containing the result of squaring each of the numbers in the given array 
+// by two 
+
+function squareTheNumbers (numbers) {
+    const array = []
+    for (let num of numbers) {
+        square = num** 2; {
```
Comment (radishmouse) [inline scripts/index.js:42]: Great solution!
I don't know that you need the curly braces, though 🤔

File: scripts/index.js [hunk 2]
```diff
@@ -1 +1,179 @@
...
+const coolCities = cities.filter(isCool);
+
+
+// write a function which takes an array of city objects and returns an array of the cities names 
+
+function returnCityNames(city) {
+    return (city["name"])
+}
+
+const cityNames = cities.map(returnCityNames);
```
Comment (radishmouse) [inline scripts/index.js:95]: Nice!

File: scripts/index.js [hunk 3]
```diff
@@ -1 +1,179 @@
...
+
+// call3Times(printsHelloWorld);
+
+
+// write a function <callNTimes> that takes two arguments: <times> as a number 
+// and <fun> as a function. it will call that function for that many times. 
+// you are allowed to use a loop in the implementation of callNTimes 
+
+function callNTimes (times, fun) {
+    for(let i = 0; i< times; i++) {
```
Comment (radishmouse) [inline scripts/index.js:144]: Awesome!

**Review by radishmouse — COMMENTED** [review]
Hooray!!! 💥
I'm so proud of you for working through the bonus exercises!

Great work!

**Pull request closed without merging**